jyotisha/panchaanga/temporal/festival/applier/solar.py

```diff
@@ -22,6 +25,23 @@ def assign_all(self):
     self.assign_agni_nakshatra()
 
 
+  def assign_sankranti_punyakaala(self):
+    PUNYA_KAALA = {1: (10, 10), 2: (16, 16), 3: (0, 60), 4: (30, 0), 5: (16, 16), 6: (0, 60),
+                   7: (10, 10), 8: (16, 16), 9: (0, 60), 10: (0, 20), 11: (16, 16), 12: (0, 60)}
```

Comment:
Would be good to put this data into a json or toml placed somewhere like adyatithi/misc_data

Reply:
This is not going to be used by any other function; so thought of keeping it local here, with reference. Will think over.

Reply:
Idea is that other ports (eg. kotlin/ js) can have ready-to-reuse data.